pkg/zfs/volume.go

```diff
@@ -63,7 +67,7 @@ var (
 func init() {
 
 	OpenEBSNamespace = os.Getenv(OpenEBSNamespaceKey)
-	if OpenEBSNamespace == "" {
+	if OpenEBSNamespace == "" && os.Getenv("OPENEBS_NODE_DRIVER") != "" {
```

Comment:
Is namespace only required for node driver?

Reply:
node-driver env can be "agent" or "controller". So it it needed if we are using ZFS-LocalPV driver.